src/pretix/static/pretixpresale/scss/main.scss

```diff
@@ -69,6 +69,17 @@ html {
     }
 }
 
+.on-focus-visible:focus, .on-focus-visible:focus-within {
+    height: auto;
+    width: auto;
+    overflow: auto;
+    clip: auto;
+}
+#skip-to-main {
+    /* padding is needed to make focus-outline visible */
+    padding: 6px;
+}
```

Comment:
If it's on-focus-visible, it probably needs a color with enough contrast to the background, doesn't it?

![image](https://github.com/user-attachments/assets/e2f6007f-9bf1-45e5-819b-257264523a03)

Reply:
Fixed it with `--pretix-link-contrast-color`. I also added a background-color to overlay the back-to-organizer-link, if the viewport is to narrow.